quickwit/quickwit-search/src/root.rs

```diff
@@ -291,6 +291,9 @@ fn simplify_search_request_for_scroll_api(req: &SearchRequest) -> SearchRequest
         snippet_fields: Vec::new(),
         // We remove the scroll ttl parameter. It is irrelevant to process later request
         scroll_ttl_secs: None,
+        // TODO unclear as to what we should do when scroll and search after are mixed. I guess
+        // honor search_after?
+        search_after: req.search_after.clone(),
```

Comment:
Different subject than your TODO, but can you open a ticket to have scroll use the superior search_after mechanics?

For the TODO: I think it is best to either return an error OR do whatever elasticsearch does.

Reply:
> Different subject than your TODO, but can you open a ticket to have scroll use the superior search_after mechanics?

there is already #3748 for just that. I actually took #3523 to unblock that ticket

Reply:
> `search_after` cannot be used in a scroll context.

this is ES behaviour.